src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/io/AprilTagPhotonCameraIO.java

```diff
@@ -26,15 +28,16 @@ public class AprilTagPhotonCameraIO extends AprilTagCameraIO {
 
     public AprilTagPhotonCameraIO(String cameraName, SimCameraProperties cameraProperties) {
         photonCamera = new PhotonCamera(cameraName);
-        cameraSim = new PhotonCameraSim(photonCamera, cameraProperties);
-        cameraSim.enableProcessedStream(true);
-        cameraSim.enableDrawWireframe(true);
+        if (CameraConstants.SHOULD_USE_CAMERA_SIMULATION && MotorSubsystem.isExtensiveLoggingEnabled()) {
+            cameraSim = new PhotonCameraSim(photonCamera, cameraProperties);
+            cameraSim.enableDrawWireframe(true);
+        } else
+            cameraSim = null;
```

Comment:
instead of else just use `return;`

Reply:
It still needs to initialize it as something

Reply:
```suggestion
            cameraSim.enableDrawWireframe(true);
            return;
        }
       cameraSim = null;
```
this is what I mean